front/src/applications/operationalStudies/types.ts

```diff
@@ -95,6 +95,7 @@ export type TrainSpaceTimeData = {
     times: number[];
   }[];
   departureTime: Date;
+  paced?: PacedTrain['paced'];
```

Comment:
It would be better to use the id to distinguish a pacedTrain from a TrainSchedule.

@emersion explained why it is better to use the id in this [comment](https://github.com/OpenRailAssociation/osrd/pull/11260#discussion_r2018287962).

Here is a suggestion for the type:
```
export type TrainSpaceTimeData = {
  name: string;
  spaceTimeCurves: {
    positions: number[];
    times: number[];
  }[];
  departureTime: Date;
  signalUpdates: ProjectPathTrainResult['signal_updates'];
} & ({ id: TrainScheduleId } | { id: OccurrenceId; paced: PacedTrainWithDetails['paced'] });
```

You can create a function to check if a TrainSpaceTimeData is a trainSchedule or not:
```
export const isTrainScheduleProjection = (
  timetableItem: TrainSpaceTimeData
): timetableItem is Extract<TrainSpaceTimeData, { id: TrainScheduleId }> =>
  isTrainSchedule(timetableItem.id);
```

Reply:
Also, you should parse use directly the type duration and not string here (PacedTrain is the type returned by editoast, PacedTrainWithDetails the type across the frontend).
```suggestion
  paced: PacedTrainWithDetails['paced'];
```

Reply:
done